examples/test_data_gen/construct_test_data_flow/score_question_prompt.jinja2

```diff
@@ -0,0 +1,7 @@
+system:
+Determine if the given question can be clearly understood even when presented without any additional context. Specify reason and verdict is a valid json format.
+
```

Comment:
update file name to "validate_question_promptflow.jinja2" and "validate_context_promptflow.jinja2"